pages/phases/getting-started/timeline.html

```diff
@@ -6,7 +6,7 @@
 sidenav: hiring-phases
 phase: getting-started
 sub-phase-order: 035
-intro: In your first use of SME-QA, target an eight week timeline from the job analysis workshop to selections. In subsequent postings of the same role, five weeks is possible. This timeline requires a project manager, HR specialist, and SMEs committed to this schedule, and hiring managers committed to making quick selections. 
+intro: Assuming you have a dedicated project manager and available HR specialist, you can aim to complete the process from posting the job ananouncement through to issuing the certificate in just 6 weeks. 
```

Comment:
i changed this to 6 weeks but also changed the timeframe to start at JOA not job analysis.